src-tauri/backend/bot/combat_mode_v2.py

```diff
@@ -0,0 +1,367 @@
+import copy
+import os
+from typing import List
+import time
+from time import sleep
+from numpy import random
+
+
+# from bot.game import Game
+from utils.settings import Settings
+from utils.message_log import MessageLog as Log
+from utils.image_utils_v2 import ImageUtils
+from utils.mouse_utils import MouseUtils
+from bot.window import Window
+
+class CombatMode:
```

Comment:
Let's rename this class to `CombatModeV2` to match its file name and to further differentiate itself from the original `CombatMode`.